<launch>

    <arg name="project" default="collect_data"/>
    
    <!-- Parameters -->
    <rosparam file="$(find collect_data)/config/data_params.yaml" command="load" />

    <arg name="episodeIndex" default="0"/>
    <arg name="datasetDir" default="/home/agilex/data"/>
    <arg name="publishFirst" default="false"/>
    <arg name="publishRate" default="30"/>
    <param name="episodeIndex" value="$(arg episodeIndex)"/>
    <param name="datasetDir" value="$(arg datasetDir)"/>
    <param name="publishFirst" value="$(arg publishFirst)"/>
    <param name="publishRate" value="$(arg publishRate)"/>

    <node pkg="$(arg project)" type="$(arg project)_dataPublish" name="$(arg project)_dataPublish" output="screen"/>

</launch>
